src/app/static/src/app/store/baseline/baseline.ts

```diff
@@ -60,7 +60,33 @@ export const baselineGetters = {
         const validOrMissingPJNZ = !state.pjnzError
         const validOrMissingPop = !state.populationError
         return validOrMissingPJNZ && validOrMissingPop && state.validatedConsistent && !!state.shape
-    }
+    },
+    selectedDatasetAvailableResources: (state: BaselineState, getters: any, rooState: DataExplorationState) => {
+        const resources: { [k in keyof DatasetResourceSet]?: DatasetResource | null } = {}
+        const { selectedDataset } = state
+
+        if (selectedDataset?.id && selectedDataset.resources) {
+            const selectedDatasetFromDatasets = rooState.adr.datasets.find(dataset => dataset.id === selectedDataset.id) || null
+
+            const resourceTypes = {
+                pjnz: "inputs-unaids-spectrum-file",
+                pop: "inputs-unaids-population",
+                shape: "inputs-unaids-geographic",
+                survey: "inputs-unaids-survey",
+                program: "inputs-unaids-art",
+                anc: "inputs-unaids-anc"
+            }
```

Comment:
We do have these types in config.properties in backend, if you can't re-use that then you should make this reusable for frontend. You can add it to util and reuse in `baseline/getters.test.ts` and here

Reply:
I've added it to the utils, but I'm unsure where/how you intended it to be reused in the getters test.